### PR TITLE
Feature/2021 update

### PR DESCRIPTION
**Pull request by SimonDarksideJ**
# Reality Collective - Reality Toolkit Pull Request

## Overview

* Updating for latest Spatial Persistence Service updates
* Updated dependencies
* Tested

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,22 @@
...
--->
-
-# Reality Toolkit - Spatial Persistence Data Provider - Azure Spatial Anchors
-
-The Spatial Persistence ASA (Azure Spatial Anchors) package for the [Reality Toolkit](https://github.com/realitycollective/com.realitytoolkit.core).
-
-[Microsoft Azure Spatial Anchors](https://azure.microsoft.com/en-us/services/spatial-anchors/#:~:text=%22Azure%20Spatial%20Anchors%20is%20the%20next%20step%20in,data%20at%20the%20right%20time%20across%20project%20phases.) implementation data provider for the Reality Toolkit [Spatial Persistence](https://github.com/realitycollective/com.realitytoolkit.spatial-persistence) System.
+[Microsoft Azure Spatial Anchors](https://azure.microsoft.com/en-us/services/spatial-anchors/#:~:text=%22Azure%20Spatial%20Anchors%20is%20the%20next%20step%20in,data%20at%20the%20right%20time%20across%20project%20phases.) vendor implementation for the Reality Toolkit [Spatial Persistence](https://github.com/realitycollective/com.realitytoolkit.spatial-persistence) System
 
 ## What's included?
```
Comment (FejZa) [inline README.md]: This should be "What is included?"

**Review by FejZa — CHANGES_REQUESTED** [review]

File: Runtime/Modules/ASASpatialPersistenceServiceModule.cs [hunk 2]
```diff
@@ -435,72 +341,128 @@ public async void DeleteAnchors(params Guid[] ids)
...
+        }
 
-        /// <inheritdoc />
-        public event Action<Guid, GameObject> AnchorLocated;
+        /// <summary>
+        /// Anchor located by the ASA Cloud watcher service, returns the ID reported by the service for the anchor via <see cref="AnchorLocated"/> event
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="args"></param>
+        private void CloudManager_AnchorLocated(object sender, AnchorLocatedEventArgs args)
```
Comment (zeratul9911) [inline Runtime/Modules/ASASpatialPersistenceServiceModule.cs]: Do you not want to use camelCase here?

**Review by zeratul9911 — COMMENTED** [review]
Some private methods in the module do not have the same naming style

File: README.md (hunk 1 above)
Comment (SimonDarksideJ) [reply README.md]: Updated

File: Runtime/Modules/ASASpatialPersistenceServiceModule.cs (hunk 2 above)
Comment (SimonDarksideJ) [reply Runtime/Modules/ASASpatialPersistenceServiceModule.cs]: Updated

**Review by FejZa — APPROVED** [review]

**Pull request merged**